Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -10157,7 +10169,9 @@ def create_indicator_result_with_dbotscore_unknown(indicator, indicator_type, re
                                             value=indicator,
                                             dbot_score=dbot_score,
                                             data={},
-                                            context_prefix=context_prefix)
+                                            context_prefix=context_prefix,
+                                            relationships=relationships
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)
```suggestion
                                            relationships=relationships,
```